test/integration/buckets-client-basic.test.js

```diff
@@ -142,6 +142,7 @@ test('buckets client basic', testOpts, function (suite) {
                                 function (err, res) {
             t.ifError(err);
             t.ok(res);
+            t.equal(res.headers['durability-level'], "2");
```

Comment:
nit: use single-quotes, ditto below

Reply:
Man, rust's got me messing this up a lot in js world. Fixed in https://github.com/joyent/node-manta/pull/384/commits/786d7f3c46b0489e117c346cf604e1e240db2dfe!